PROVISIONING-METAL.md

```diff
@@ -0,0 +1,187 @@
+  * `enabled` (boolean, required): Enables DHCP6.
+  * `optional` (boolean): the system will request a lease using this protocol, but will not wait for a valid lease to consider this interface configured.
+
+Example `net.toml` with comments:
+```toml
+version = 1
+
+# "eno1" is the interface name
+[eno1]
+# Users may turn on dhcp4 and dhcp6 via boolean
+dhcp4 = true
+dhcp6 = true
+primary = true
+
+# "eno2" is the second interface in this example
+[eno2.dhcp4]
+# `enabled` is a boolean and is a required key when
+# setting up DHCP this way
+enabled = true
+# Route metric may be supplied for ipv4
+route-metric = 200
+
+[eno2.dhcp6]
+enabled = true
+optional = true
+```
+
+**An additional note on network device names**
+
+Interface name policies are [specified in this file](https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/release/80-release.link#L6); with name precedence in the following order: onboard, slot, path.
+Typically on-board devices are named `eno*`, hot-plug devices are named `ens*`, and if neither of those names are able to be generated, the “path” name is given, i.e `enp*s*f*`.
+
+### Boot Configuration
+
+Bottlerocket for bare metal uses a feature of the Linux kernel called [Boot Configuration](https://www.kernel.org/doc/html/latest/admin-guide/bootconfig.html), which allows a user to pass additional arguments to the kernel command line at runtime.
+An immediate use of this feature for most users is setting `console` settings so boot messages can be seen on the appropriate consoles.
+
+In order to make use of this feature, an initrd is created with the desired settings encoded inside it.
+The initrd is empty save for the encoded boot config data.
+To create the initrd, you must first create a configuration file containing key value pairs for the settings you would like to pass to kernel / init.
+Full syntax is described in the [Boot Config documentation](https://www.kernel.org/doc/html/latest/admin-guide/bootconfig.html#config-file-syntax), but a simple example is provided below that shows the format of console settings as well as an example `systemd` parameter.
+
+The two acceptable prefixes to settings are `kernel` and `init`.
+Settings prefixed with `kernel` are added to the beginning of the kernel command line.
+Settings prefixed with `init` are added to the kernel command line after the `--`, but before any existing init parameters.
+
+In the example below, two console devices are set up, and `systemd`'s log level is set to `debug`.
+
+Example Boot Configuration:
+```
+kernel {
+    console = tty0, "ttyS1,115200n8"
+}
+init {
+    systemd.log_level = debug
+}
+```
+
+The Bottlerocket SDK provides the `bootconfig` CLI tool, which is used to create a Boot Configuration initrd.
+To create the Boot Configuration initrd, create a config file named `bootconfig-input` containing your desired key/value pair kernel and init arguments.
+
+Then run the following (you will need Docker installed):
```

Comment:
Should we also mention the related `cargo make` tasks as another option in addition to the `docker run` commands?